Makefile

```diff
@@ -172,6 +180,8 @@ release: fmtcheck embed_bundle build_docs_pdf
 
 	pushd $(RELEASE_DIR) && sha256sum * > sha256sum.txt && popd
 
+	@mv $(RELEASE_INFO) $(RELEASE_DIR)/$(RELEASE_INFO)
```

Comment:
@anjannath can you move this to line number 180 so that we can create the shasum for this file also?